docs/guides/anomaly-detection-configuration/fail_on_zero.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "fail_on_zero"
```

Comment:
This setting should only be applicable to volume anomalies, and needs to also appear in the volume anomalies parameter list [here](https://docs.elementary-data.com/guides/anomaly-detection-tests/volume-anomalies#test-configuration)